debugtools/DDR_VM/src/com/ibm/j9ddr/AuxFieldInfo29.dat

```diff
@@ -263,6 +263,7 @@ J9JavaVM.jitConfig = required
 J9JavaVM.jniGlobalReferences = required
 J9JavaVM.jniWeakGlobalReferences = required
 J9JavaVM.jvmtiData = required
+J9JavaVM.liveVirtualThreadList = required
```

Comment:
does this need to be required if is it only used through MH access? same for virtualThreadLinkNextOffset

Reply:
indeed, these are not needed